src/main/groovy/org/broadinstitute/orsp/IssueInfo.groovy

```diff
@@ -0,0 +1,12 @@
+package org.broadinstitute.orsp
+
+class IssueInfo {
```

Comment:
Nit: naming - this is very specific to what OSAP is looking for, I would suggest a more OSAP specific name.

Reply:
Agree. could it be OSAPDataFeed ?

Reply:
Works for me 👍